lsp-ui-doc.el

```diff
@@ -771,8 +820,8 @@ BUFFER is the buffer where the request has been made."
 
 (defun lsp-ui-doc-hide-frame-on-window-change (fun window &optional no-record)
   "Delete the child frame if currently selected window changes.
-Does nothing if the newly-selected window is the same window as
-before, or if the new window is the minibuffer."
+  Does nothing if the newly-selected window is the same window as
+  before, or if the new window is the minibuffer."
```

Comment:
May I ask why you are indenting docstring over here? 😕 

Thanks!

Reply:
Not sure, you're right it shouldn't be there